### DISCUSSION TITLE
How to exclude certain custom task statuses from the dataview filter?

### DISCUSSION THREAD

**Discussion by claremacrae** [ivan-lednev/obsidian-day-planner#498] [Q&A]
### Goal

I use a couple of different status characters to represent Cancelled tasks, so it would be helpful to be able to exclude them from the dat planner.

In dataview, I can exclude them with the filter `status != "-" and status != ">"`.

For exemple:

~~~
```dataview
TASK
WHERE !fullyCompleted and status != "-" and status != ">"
LIMIT 200
```
~~~

### What I tried

I tried adding this to the Dataview source:

```
"/" and status != "-" and status != ">"
```

And get this error - and realised that its prompt is specific about the types of filters allowed, and this is not one of them:

![image](https://github.com/user-attachments/assets/855ca5df-d9b4-4692-8a7f-9dda1118e414)

### Is there any other way?

So, is there any way to specify which statuses to exclude from the Planner please?

**Comment by ivan-lednev** [comment]
Hello, right now there is no way to handle different task statuses. The Dataview filter works at the level of files, which means file paths and tags.

**Reply by claremacrae** [reply]
Understood. I ended up making this (hard-coded) change to my fork for now...

https://github.com/ivan-lednev/obsidian-day-planner/commit/fdda16c109a13efd8019686cde38e5bbfe6bfbc5
